plugins/hikvision/src/main.ts

```diff
@@ -457,7 +482,7 @@ export class HikvisionCamera extends RtspSmartCamera implements Camera, Intercom
         return false;
     }
 
-    updateDevice() {
+    async updateDevice() {
```

Comment:
this should not be async, it should update the device with the current known state. figure out ptz state prior to calling this.